static/_redirects

```diff
@@ -164,6 +164,7 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
 # /docs/enterprise/about uses multiple redirects
 /docs/enterprise/about /docs/releases/enterprise
 /docs/releases/enterprise /docs/releases/enterprise/about
+/docs/releases/enterprise/about /docs/enterprise
```

Comment:
Would it make sense to simplify some of these redirects while you're here?

I think this line makes the redirect from /docs/enterprise → /docs/releases/enterprise/about on line 151 above redundant, so we could delete line 151.

And maybe a lot of this section could be simplified by removing `/releases` from the right hand side? (like on lines 156-162)

Reply:
That works. I need to comb through this whole file to remove redundant links.